index.js

```diff
@@ -25,6 +25,9 @@ function mewt(target) {
       return mewt(newObj)
     },
     $unset: prop => {
+      if (isA && Number.isInteger(prop)) {
+        return mewt([...target.slice(0, prop), ...target.slice(prop + 1)])
```

Comment:
Sure but that leaves as with double results again since it returns the removed part :( If you want it shorter maybe `target.filter((_, i) => i !== prop)`?